### DISCUSSION TITLE
This should be below 5 seconds to minimize risk of losing rewards. 2021-05-12T17:53:46.190 full_node chia.full_node.full_node: WARNING  Wasn't able to add transaction with id 5e0fd02fa737a16dd9252f00973c1de56ebd70a947592925f1389673e4a78c2a, status 3 error: Err.COIN_AMOUNT_NEGATIVE

### DISCUSSION THREAD

**Discussion by RRCHA000** [Chia-Network/chia-blockchain#6231] [Support]
Do I need to improve internet connection or is there something I can do to reduce the time to less than 5 seconds?

**Comment by XIII39** [comment]
> This should be below 5 seconds to minimize risk of losing rewards

This message usually comes up when you are copying plots to the drive. Active I/O on the drive is high so the look time is slow. Once you finish copying the plot file, the error should go away

**Comment by BrandtH22** [comment]
@RRCHA000 >, since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
